compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmSymbols.kt

```diff
@@ -221,7 +221,7 @@ class JvmSymbols(
                 addValueParameter("completion", continuationType.makeNullable())
             }
             klass.addFunction("invokeSuspend", irBuiltIns.anyNType, Modality.ABSTRACT).apply {
-                addValueParameter("result", resultClassStub.typeWith(irBuiltIns.anyNType))
+                addValueParameter("\$result", resultClassStub.typeWith(irBuiltIns.anyNType))
```

Comment:
I somehow missed it. Thanks!
Minor: there is `SUSPEND_CALL_RESULT_NAME`, which, in fact, I should've used.